### ISSUE TITLE
Space Bar Not Working on Virtual Keyboard Android Tablet

### ISSUE THREAD

**Issue by ABION-Technology** [termux/termux-packages#5119]
I have several tablets, an iPad, a Samsung Galaxy Tab Android tablet, and an RCA Voyager Android tablet. On the RCA Voyager Android Tablet (Android 8), the space bar on the virtual keyboard will not work, not sure of the cause of this, went in to settings and it is using Gboard keyboard, but do not see any setting related to the space bar on virtual keyboard. Has anyone else encountered this? and if so, what was the fix? tried rebooting also, but that did not fix.

**Comment by FreddieOliveira** [comment]
Have you tried using other keyboards, like AnySoftKeyboard or hacker's keyboard?

**Comment by ABION-Technology** [comment]
No, just the virtual keyboard, i dont have some bluetooth keyboards but I
prefer to use the virtual one.

Thanks.

*Michael G. Workman*
(321) 432-9295
michael.g.workman@gmail.com

**Comment by ABION-Technology** [comment]
Actually I meant to type I DO HAVE SOME BLUETOOTH KEYBOARDS,

But I like using the virtual keyboard on my small RCA Voyager android
tablet, it is only a 7" tablet.

*Michael G. Workman*
(321) 432-9295
michael.g.workman@gmail.com

**Comment by Grimler91** [comment]
Hacker's keyboard is another keyboard app: https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard
Try it or any other virtual keyboard app. There's probably some problem with the default one on your device

**Comment by ABION-Technology** [comment]
Ok great, thanks for the suggestion, I will check it out.

My RCA Voyager Android Tablet is a low cost one, only $50, so not really
surprised about the bug with the keyboard.

Termux is a very cool app, I like it very much.

*Michael G. Workman*
(321) 432-9295
michael.g.workman@gmail.com
